FRCore/FRCore/Keychain/SecuredKey.swift

```diff
@@ -60,7 +60,7 @@ public struct SecuredKey {
     
     /// Initializes SecuredKey object with designated service; SecuredKey may return nil if it failed to generate keypair
     /// - Parameter applicationTag: Unique identifier for SecuredKey in Keychain Service
-    public init?(applicationTag: String, accessGroup: String? = nil) {
+    public init?(applicationTag: String, accessGroup: String? = nil, kSecAttrAccessibleFlag: KeychainAccessibility) {
```

Comment:
I would suggest make the parameter anme match the one in the ```KeychainOptions```. Meaning rename it from ```kSecAttrAccessibleFlag``` to ```accessibility```.

Also, I am wondering if instead of forcing the developer to pass the ```accessibility``` parameter, we set the default value to ```.afterFirstUnlock```. Aligning with the behaviour we see on the ```KeychainOptions```.

Reply:
I agree with Rodrigo. We should preserve the old interface, otherwise, this becomes a "breaking" change...

Reply:
btw, because of this interface change, some of the test cases in FRCore do not compile and make the Jenkins build fail... 
The tests in question are in here: /FRCore/FRCoreTests/FRCore/Util/SecuredKeyTests.swift

They all pass once I pass "kSecAttrAccessibleFlag: .afterFirstUnlock" as parameter...